pkg/component/kubernetes/controllermanager/controllermanager_test.go

```diff
@@ -202,98 +173,40 @@ var _ = Describe("KubeControllerManager", func() {
 			return pdb
 		}
 
-		hvpaFor = func(config *HVPAConfig) *hvpav1alpha1.Hvpa {
-			scaleDownUpdateMode := config.ScaleDownUpdateMode
-			if scaleDownUpdateMode == nil {
-				scaleDownUpdateMode = ptr.To(hvpav1alpha1.UpdateModeAuto)
-			}
-
-			return &hvpav1alpha1.Hvpa{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:      hvpaName,
-					Namespace: namespace,
-					Labels: map[string]string{
-						"app":  "kubernetes",
-						"role": "controller-manager",
-						"high-availability-config.resources.gardener.cloud/type": "controller",
+		vpaFor = func(isScaleDownDisabled bool) *vpaautoscalingv1.VerticalPodAutoscaler {
+			vpa := &vpaautoscalingv1.VerticalPodAutoscaler{
+				ObjectMeta: metav1.ObjectMeta{Name: vpaName, Namespace: namespace, ResourceVersion: "1"},
+				Spec: vpaautoscalingv1.VerticalPodAutoscalerSpec{
+					TargetRef: &autoscalingv1.CrossVersionObjectReference{
+						APIVersion: "apps/v1",
+						Kind:       "Deployment",
+						Name:       v1beta1constants.DeploymentNameKubeControllerManager,
```

Comment:
```suggestion
						Name:      "kube-controller-manager",
```

Reply:
Done.
FYI, there was already [preexisting test code using the constant](https://github.com/gardener/gardener/blob/d7e2ecb01961344464c0f3c70fd5d4e8f69bd8d6/pkg/component/kubernetes/controllermanager/controllermanager_test.go#L276).